config.go

```diff
@@ -80,6 +80,11 @@ type config struct {
 	// InstrumentAttributesGetter will be called to produce additional attributes while recording metrics to instruments.
 	// Default returns nil
 	InstrumentAttributesGetter InstrumentAttributesGetter
+
+	// DisableSkipErrMetrics will skip the recoding of any SkipErr metric if enabled.
+	// Since SkipErr results in a retry, the actual database interaction will still be recorded.
+	// Default is false
+	DisableSkipErrMetrics bool
```

Comment:
```suggestion
	// DisableSkipErrMetrics will skip the recoding of the driver.ErrSkip errors as measurements if enabled.
	// Since driver.ErrSkip results in a retry, and the actual database interaction will still be recorded.
	// Default is false
	DisableSkipErrMeasurement bool
```
I feel `Measurement` is more adhesive with https://opentelemetry.io/docs/specs/otel/metrics/api/

Reply:
Updated as suggested